README.adoc

```diff
@@ -184,22 +183,41 @@ credentials-secret-access-key:<your-secret_access_key>
 credentials-session-token:<your-session_token>
 ----
 
-=== First Steps
+== Metrics
+
+The S3 cluster discovery extension delivers a set of metrics that can be used to monitor the behavior in a dashboard.
+
+The following metrics are available:
+
+These two counter metrics indicate a successful or failed S3 query attempt in order to receive the IP addresses of cluster members:
+
+----
+com.hivemq.dns-cluster-discovery-extension.query.success.count
```

Comment:
```suggestion
com.hivemq.extensions.cluster.discovery.s3.query.success.count
```

Reply:
Why was this resolved?
If these are new metrics we could follow a proper schema this time.

Reply:
We created an ADR for this specific topic:
https://shiny-couscous-d03eece6.pages.github.io/log4brains/adr/2022-07-18-extension-metricnaming/